test

```diff
@@ -181,9 +181,13 @@ function release_pass {
 	rm -f ./bin/etcd-last-release
 	# to grab latest patch release; bump this up for every minor release
 	UPGRADE_VER=$(git tag -l --sort=-version:refname "v3.2.*" | head -1)
-	if [ -n "$MANUAL_VER" ]; then
+	if [[ ${MANUAL_VER} ]]; then
```

Comment:
why doesn't `-n` work? If ${MANUAL_VER} is an empty string then the condition should evaluate to false...
```
emptyStr=""
if [ -n "$emptyStr" ]; then
        echo non-empty
else
        echo empty
fi
```